README.md

```diff
@@ -25,17 +25,21 @@ In an earlier lesson, we learned about truthy and falsey values in JavaScript.
 The logical NOT operator (`!`), also called the _bang operator_, operates on an
 expression, returning the opposite of the expression's truthiness. If `x`
 resolves to a truthy value, `!x` returns `false`. If `x` is falsey, `!x` returns
-`true`:
+`true`. Let's add the following code into our REPL and see what the values are.
 
-<iframe height="400px" width="100%" src="https://repl.it/@LizBurton/UniqueAnotherAutomaticvectorization?lite=true" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
+```js
+const truthyValue = 'This value is truthy.';
+const falseyValue = 0;
+
+!truthyValue;
+```
 
-If you click the Run button you'll see the bang operator in action, returning
-the reverse of `truthyValue`'s truthiness. Then modify the code on line 4 to
-verify that it works for the falsey value as well.
+<iframe height="400px" width="100%" src="https://replit.com/@lizbur10/Sandbox?lite=1&outputonly=1" scrolling="no" frameborder="no" allowtransparency="true" allowfullscreen="true" sandbox="allow-forms allow-pointer-lock allow-popups allow-same-origin allow-scripts allow-modals"></iframe>
 
-> **Reminder:** Recall that you will need to click "open in repl.it" in the
-> upper right corner of the REPL window to open it in a new browser tab, then
-> sign in to your repl.it account before you can fork the code and modify it.
+Verify the above code works by running it in the REPL console (remember to hit
+`enter`!) Play around with it and you'll see the bang operator in action,
+returning the reverse of `truthyValue`'s truthiness. Then modify the code on the
+last line to verify that it works for `falseyValue` as well.
```

Comment:
Let's say something like "Be sure to verify that it works for `falseyValue` as well" instead - the "modify the code" phrase seems a little off without a code editor.